tasks/wdl/ImputationBeagleQcTasks.wdl

```diff
@@ -26,6 +26,7 @@ task QcChecks {
         line_count=$(bcftools stats ~{vcf_input}  | grep "number of records:" | awk '{print $6}')
         if [ "$line_count" -gt 10000000 ]; then
             echo "Greater than 10 million variants found in input VCF." >> qc_messages.txt
+            echo "false" > passes_qc.txt
```

Comment:
ahhh right, thanks for catching this!!

Reply:
thanks to tests for catching it!

Reply:
im acutally not sure how this works when first setting up tests for a pipeline.  It looks like the automated tests arent working yet presumably because we havent set up the truth yet.  How did tests catch this bug then?